tests/integration/conftest.py

```diff
@@ -0,0 +1,76 @@
+from logging import getLogger
```

Comment:
Missing:
- copyright + license header  
- return types and docstrings in most of the fixtures